.github/actions/orca/orca-pytest-ray-ctx-py37-spark3-action/nightly-test/action.yml

```diff
@@ -0,0 +1,42 @@
+name: 'Run Orca Python Ray Ctx Py37 Spark3'
+description: 'Run Orca Python Ray Ctx Py37 Spark3'
+runs:
+  using: "composite"
+  steps:
+    - name: Run Test
+      shell: bash
+      run: |
+        source activate py37
+        export _JAVA_OPTIONS="-XX:MaxPermSize=3G -Xmx10G"
+        export MAVEN_OPTS="-XX:ReservedCodeCacheSize=512m -XX:MaxPermSize=3G -Xmx10G"
+        export http_proxy="http://child-prc.intel.com:913/"
+        export https_proxy="http://child-prc.intel.com:913/"
+        
+        export JAVA_HOME=/opt/work/jdk8
+        export CLASSPATH=.:${JAVA_HOME}/lib:${JAVA_HOME}/jre/lib:${JAVA_HOME}/lib/tools.jar:${JAVA_HOME}/lib/dt.jar
+        export PATH=${JAVA_HOME}/bin/:${JAVA_HOME}/jre/bin:${PATH}
+        export PATH=/opt/work/apache-maven-3.6.3/bin:$PATH 
```

Comment:
do we need to write these here?

Reply:
The ctx runner may have different environment variables, just keep them unchanged.